pkg/ddl/backfilling.go

```diff
@@ -636,12 +647,7 @@ func makeupDecodeColMap(dbName model.CIStr, t table.Table) (map[int64]decoder.Co
 	return decodeColMap, nil
 }
 
-var backfillTaskChanSize = 128
-
-// SetBackfillTaskChanSizeForTest is only used for test.
```

Comment:
Will this remove operation cause test case execution slower？

Reply:
No, the imact is negligible.